File: trunk/unityClientSample/Assets/src/protocol/SurrenderRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 我选择死亡 协议:27
    /// </summary>
	[Proto(value=27,description="我选择死亡")]
	[ProtoContract]
	public class SurrenderRequest
	{
        /// <summary>
        ///  roomId
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string roomId;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/battle/SurrenderRequest.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;


@Data
@Proto(value = 27, description = "我选择死亡")
public class SurrenderRequest
{
	@Protobuf(description = "roomId")
	private String roomId;
}
